central/src/components/CreateQuestion.jsx

```diff
@@ -97,12 +96,24 @@ export default function QuestionForm({ updateQuestion, question: initialState, g
       window.alert("Please enter a question");
       return;
     }
-
```

Comment:
nice job making use of the existing window.alert functionality to catch more errors. This will help a lot when we go coordinate this with U/X